boards/common/bflb_mcu_tool.board.cmake

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
It looks like this board is not covered by any of the areas of maintenance.

Reply:
Fixed